client/wfprev-war/src/main/angular/src/app/components/create-new-project-dialog/create-new-project-dialog.component.html

```diff
@@ -0,0 +1,113 @@
+              class="project-input"
+            />
+            <div class="error" *ngIf="projectForm.get('projectName')?.hasError('maxlength') && projectForm.get('projectName')?.touched">
+              Exceeds number of allowed characters
+            </div>
+          </div>
+          <div class="form-field">
+            <label>Lat/Long</label>
+            <input type="text" formControlName="latLong" placeholder="Manually entered or pasted from clipboard" class="project-input"/>
+            <div class="error" *ngIf="projectForm.get('latLong')?.hasError('maxlength') && projectForm.get('latLong')?.touched">
+              Exceeds number of allowed characters
+            </div>
+          </div>
+          <div class="form-field">
+            <label>Business Area<span class="required">*</span></label>
+            <select formControlName="businessArea">
+              <option value="">select...</option>
+              <option *ngFor="let area of businessAreas" [value]="area">{{ area }}</option>
+            </select>
+          </div>
+        </div>
+        <div class="form-row">
+          <div class="form-field">
+            <label>Forest Region<span class="required">*</span></label>
+            <select formControlName="forestRegion">
+              <option value="">select...</option>
+              <option *ngFor="let region of forestRegions" [value]="region">{{ region }}</option>
+            </select>
+          </div>
+          <div class="form-field">
+            <label>Forest District<span class="required">*</span></label>
+            <select formControlName="forestDistrict">
+              <option value="">select...</option>
+              <option *ngFor="let district of forestDistricts" [value]="district">{{ district }}</option>
+            </select>
+          </div>
+          <div class="form-field">
+            <label>BC Parks Region<span class="required">*</span></label>
+            <select formControlName="bcParksRegion">
+              <option value="">select...</option>
+              <option *ngFor="let region of bcParksRegions" [value]="region">{{ region }}</option>
+            </select>
+          </div>
+          <div class="form-field">
+            <label>BC Parks Section<span class="required">*</span></label>
+            <select formControlName="bcParksSection">
+              <option value="">select...</option>
+              <option *ngFor="let section of bcParksSections" [value]="section">{{ section }}</option>
+            </select>
+          </div>
+        </div>
+        <div class="form-row">
+          <div class="form-field">
+            <label>Project Lead</label>
+            <input type="text" formControlName="projectLead" placeholder="Last Name, First Name" class="project-input"/>
+            <div class="error" *ngIf="projectForm.get('projectLead')?.hasError('maxlength') && projectForm.get('projectLead')?.touched">
+              Exceeds number of allowed characters
```

Comment:
should we be putting the text from these error message into a shared source so that if we want to change the language, it's a single change vs. across multiple components?  more of a question than a statement.

Reply:
Sounds good, I haved added a messages.ts as shared file to use in different components